skymp5-server/cpp/papyrus_vm_lib/Reader.cpp

```diff
@@ -480,12 +445,12 @@ uint64_t Reader::Read64_bit()
   return temp;
 }
 
-std::string Reader::ReadString(int size)
+std::string Reader::ReadString(int Size)
```

Comment:
Variable names should start with lowercase
https://github.com/skyrim-multiplayer/skymp/blob/main/docs/docs_cpp_code_guidelines.md#variables

Reply:
@Pospelove Hmm, probably CI can find such inconsistencies. But it didn't find it.